test/test-fs.c

```diff
@@ -1691,6 +1691,18 @@ TEST_IMPL(fs_access) {
   ASSERT_EQ(1, access_cb_count);
   access_cb_count = 0; /* reset for the next test */
```

Comment:
Unrelated to this change, but it looks like there may be a missing `uv_fs_req_cleanup()` call here.

Reply:
I think the call is in the callback function `access_cb`.